plugins/test/unit/plugins/distributors/yum/metadata/test_updateinfo.py

```diff
@@ -6,6 +6,7 @@
 import mock
 
 from pulp.common.compat import unittest
+from pulp.server.exceptions import PulpCodedException
```

Comment:
There should be a blank line below this one for pep8 separation of import sections.